tf_agents/agents/ppo/ppo_agent.py

```diff
@@ -487,7 +492,10 @@ def _train(self, experience, weights):
         experience.observation, experience.step_type, policy_state=policy_state)
     value_preds = tf.stop_gradient(value_preds)
 
-    valid_mask = ppo_utils.make_timestep_mask(next_time_steps)
+    if self._train_only_on_full_episodes:
+      valid_mask = ppo_utils.make_timestep_mask(next_time_steps)
+    else:
+      valid_mask = tf.ones(next_time_steps.step_type.shape)
```

Comment:
is there a way to make valid_mask None here and respect that throughout the rest of the code, thereby avoiding an extra useless tensor?